package.json

```diff
@@ -43,7 +44,9 @@
     "compile-base-ui": "node ./.storybook/base-ui-compiler.js",
     "storybook:build": "build-storybook -s ./src/assets",
     "generate-stylable-index": "stc --srcDir=\"./dist/src/components\" --diagnostics --indexFile=index.st.css",
-    "generate": "wuf generate",
+    "generate": "wuf generate --codemods .wuf/generator/codemods --templates .wuf/generator/templates",
+    "postgenerate": "npm run update-components",
+    "update-components": ".wuf/update-components.sh",
```

Comment:
Super picky comment: Maybe this should be called `generate-testkits`? or something like that?

Reply:
good comment :)

maybe it should, but looking at future this will do more than generate testkits so i chose this name. also, it's same as in wsr, so maybe consistency.

just so you know, i don't like the `update-components.sh`, it should be just `wuf update`. But for now, multiple commands are needed and i didn't want to add them all to `package.json`

i can name it `generate-testkits` if you want, tell me :)

Reply:
I agree, consistency at this point might be more important.
and +1 for `wuf update`